src/hub.html

```diff
@@ -17,6 +17,13 @@
     <% } else { %>
         <script src="https://cdn.rawgit.com/aframevr/aframe/3e7a4b3/dist/aframe-master.js" integrity="sha384-EaMOuyBOi9ERV/lVDwQgz/yFWBDWPsIju5Co6oCZZHXuvbLBO81yPWn80q0BbBn3" crossorigin="anonymous"></script>
     <% } %>
+
+    <script>
+        // HACK this has to run after A-Frame but before our bundle, since A-Frame blows away the local storage setting
+        if (/[&?]debug=true/.test(document.location.search) && typeof(localStorage) !== 'undefined') {
```

Comment:
`typeof(localStorage)` -> `typeof localStorage`

Reply:
generally better to just use [`window.location`](https://www.w3.org/TR/html50/browsers.html#the-location-interface), as opposed to `document.location`

Reply:
interesting, why is that?